Cargo.toml

```diff
@@ -92,6 +96,7 @@ tokio = { version = "1.0", default-features = false, features = [
 tracing = "0.1.40"
 tracing-subscriber = "0.3"
 tz-rs = { version = "0.6" }
+typespec_derive = { path = "sdk/typespec/typespec_derive" }
```

Comment:
Let's add this up around line 30 so it's with the other first-party (in-repo) dependencies for easy reference.